src/test/java/org/folio/circulation/infrastructure/storage/inventory/ItemRepositoryTests.java

```diff
@@ -108,7 +110,7 @@ private ItemRepository createRepository(CollectionResourceClient itemsClient) {
 
     return new ItemRepository(itemsClient, locationRepository,
       materialTypeRepository, instanceRepository,
-      holdingsRepository, loanTypeRepository);
+      holdingsRepository, loanTypeRepository, circulationItemRepository);
```

Comment:
There are no new tests for scenarios with circulation items. Shouldn't we mock this new endpoint, add some circulation items and see how out code behaves when it doesn't find a regular item but finds circulation item instead?

Reply:
Added unit test